packages/backend/src/projectAdapters/dbtBaseProjectAdapter.ts

```diff
@@ -88,7 +88,7 @@ export class DbtBaseProjectAdapter implements ProjectAdapter {
 
         // Validate models in the manifest - models with invalid metadata will compile to failed Explores
         const models = Object.values(manifest.nodes).filter(
-            (node: any) => node.resource_type === 'model',
+            (node: any) => node.resource_type === 'model' && node.meta !== null,
```

Comment:
This is for a specific case when creating a project connection when the model node has `meta` as null. We still haven't been able to reproduce this.